src/schema/organization/resolvers.js

```diff
@@ -1,8 +1,28 @@
 import fetchData from '../../github/dataFetcher';
+import fetchRestData from '../../github/restDataFetcher';
 import githubQueries from '../../github/queries';
 
 export default {
   Query: {
+    commits: async (parent, args, { token }) => {
+      const login = parent.login;
+      const url = `https://api.github.com/search/commits?q=is:public+org:${login}`;
+      const data = await fetchRestData(url);
+
+      return data.total_count;
```

Comment:
Keeping the camelCase pattern, it should be **totalCount** instead of the snake_case **total_count**

Reply:
@jadsonluan *total_count* is what the API returns for me, so there isn't a way to change it